cmd/ak/common/flags.go

```diff
@@ -40,15 +41,13 @@ func ToExitCodeError(err error, what string) error {
 	var code int = GenericFailure
 	switch {
 	case errors.Is(err, sdkerrors.ErrNotFound):
-		msg = fmt.Sprintf("%s not found", what)
-		code = NotFoundExitCode
+		return NewExitCodeError(NotFoundExitCode, fmt.Errorf("%s not found", what))
 	case errors.Is(err, sdkerrors.ErrFailedPrecondition):
 		msg = fmt.Sprintf("on %s", what)
 		code = FailedPrecondition
-	case errors.As(err, resolver.NotFoundErrorType) {
-		msg = fmt.Sprintf("%s not found", what)
-		code = NotFoundExitCode
-    }
+	case errors.As(err, resolver.NotFoundErrorType):
```

Comment:
we have both sdk and resolver not found errors ?

Reply:
yup. I just moved the code. No new logic introduction